src/inmanta/protocol/ipc_light.py

```diff
@@ -35,11 +36,22 @@ class ConnectionLost(IPCException):
     pass
 
 
+ServerContext = typing.TypeVar("ServerContext")
+R = typing.TypeVar("R")
```

Comment:
It was not immediately clear to me this `R` represents the return value of the method. I think it would be good to give this TypeVar a better name. I think we should in general prevent single character identifiers, unless it's a very locally scoped loop variable.